projects/storefrontlib/cms-components/myaccount/my-coupons/my-coupons.component.ts

```diff
@@ -66,7 +67,8 @@ export class MyCouponsComponent implements OnInit, OnDestroy {
 
   constructor(
     protected couponService: CustomerCouponService,
-    protected myCouponsComponentService: MyCouponsComponentService
+    protected myCouponsComponentService: MyCouponsComponentService,
```

Comment:
MAJOR: Changes in constructor are breaking for customers. Please use `inject()` instead.

Reply:
thanks, fixed.